bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/model/equipment/AlarmSystem.java

```diff
@@ -22,6 +22,6 @@
  * @author Generated from generateTagClasses.groovy - Initial contribution
  */
 @NonNullByDefault
-@TagInfo(id = "Equipment_AlarmSystem", label = "Alarm System", synonyms = "Alarm Systems", description = "")
+@TagInfo(id = "Equipment_AlarmSystem")
```

Comment:
If we add the "additional information" like label in the provider, can't we omit these classes and also add them dynamically?

Reply:
Yes sir, it is possible to make them all created at runtime. I am sure even if I did not test it.
But maybe we should limit the number of classes created at runtime ? That was my reason to keep them as static.

Reply:
I don't think this is an issue. Once they are loaded, they are there. If they are loaded from a bundle or created at runtime makes no difference. The only thing to keep in mind is that we can't easily unload classes, so we should make sure that we don't create tons of unnecessary classes.

Reply:
I have something working with classes created at runtime but it broke most of the unit tests (for example because code in unit tests is using Bathroom.class). I will have to update unit tests.